Datastore/src/DatastoreClient.php

```diff
@@ -1161,7 +1161,7 @@ public function gqlQuery($query, array $options = [])
      *           {@see Google\Cloud\Datastore\Entity}.
      *     @type string $readConsistency See
      *           [ReadConsistency](https://cloud.google.com/datastore/reference/rest/v1/ReadOptions#ReadConsistency).
-     *     @type timestamp $readTime Reads entities as they were at the given timestamp.
+     *     @type \Google\Protobuf\Timestamp $readTime Reads entities as they were at the given timestamp.
```

Comment:
I checked that adding just `@type Timestamp` refers to Core timestamp rather than protobuf. So, this seems like a better solution.